packages/whosmysanta-backend/.eslintrc.js

```diff
@@ -31,5 +31,10 @@ module.exports = {
     'max-len': ['error', {code: 80}],
     'no-console': 'error',
     'object-curly-spacing': ['error', 'never'],
+    'space-before-function-paren': ['error', {
+      anonymous: 'always', // const foo = function () {}
+      named: 'always', // function foo () {} (Airbnb rule change)
```

Comment:
Airbnb has inconsistent styles with this rule, which is strange to experience in a codebase. I would suggest that we aim for consistency.

But yes, aside from small exceptions, sticking to Airbnb style as much as possible is also :thumbsup: from me.